docs/en/user-manual/post_sync_connector.md

```diff
@@ -0,0 +1,213 @@
+
+Post-sync connectors are executed after the main sync process completes. In the course of the sync process,
+the Post-Sync Manager marshals the following data to be used be each post-sync connector.
+
+* Identity source data, including any additional attributes
+* The current state of all users in the Admin Console after the completion of the sync process (with respect to `--adobe-users`
+  and any exclusion rules that may be specified)
+
+This post-sync data is provided to each post-sync connector for post-sync execution. It will not be altered by any post-sync
+connector.
+
+During post-sync execution, each post-sync connector will be run in the order defined in `post_sync.modules`.
+
+# Sign Sync Connector
+
+The **Adobe Sign Sync** connector introduces an optional sync workflow to manage Adobe Sign users.
+
+* Assign Sign users to specific Sign groups
+* Define Group Admin and Account Admin role assignmet rules
+
+## Native Sign &lt;-&gt; Admin Console Connection
+
+The Adobe Sign Sync connector is not required to provision users to Adobe Sign. Users assigned to Sign Enterprise plans
+are automatically granted basic Sign accounts when they are first provisioned to a Sign product profile in the
+Admin Console. When Sign users are created in this manner, they are assigned to the **Default Group** in Sign
+and are assigned **Normal User** privileges.
+
+Taking advantage of this native sync functionality with the User Sync Tool is easy - just target an Adobe Sign
+product profile in your group mapping in `user-sync-config.yml`.
+
+```yaml
+directory_users:
+  groups:
+    - directory_group: adobe-sign-enterprise
+      adobe_groups:
+        - Adobe Sign
+```
+
+Updates to user information (First Name, Last Name, and email address) are automatically synced to Sign
+the next time the user logs into Sign.
+
+## Feature Overview
+
+
+| Feature | Notes |
+|---|---|
+| Sign group assignment | Assign Sign users to a non-default group |
+| Admin Roles | Provision users in Sign with Group Admin or Account Admin privileges |
+| Automatic group creation | Sign groups are created automatically if they don't already exist |
+| Console-driven provisioning | Group and role assignment is governed by Admin Console group membership and/or Console admin role assignment |
+{: .bordertablestyle }
+
+## Configuration Overview
+
+The Adobe Sign Sync configuration file, `connector-sign-sync.yml`, defines the behavior of the Adobe Sign sync process.
+
+### Example
+
+```yaml
+# connector-sign-sync.yml
+sign_orgs:
+  - host: api.echosign.com
+    key: [API key]
+    admin_email: signadmin@example.com
+
+user_groups:
+  - Sign Users
+  - More Sign Users
+
+entitlement_groups:
+  - Adobe Sign
+
+identity_types:
+  - federatedID
+
+admin_roles:
+  - sign_role: ACCOUNT_ADMIN
+    adobe_groups:
+      - Adobe Sign Account Admins
+  - sign_role: GROUP_ADMIN
+    adobe_groups:
+      - Adobe Sign Group Admins
+```
+
+### Config Spec
+
+| key | type | required? | notes |
```

Comment:
A clearer explanation of what the groups keys mean and how they are related would be helpful. In addition, some discussion of what it means to use multiple sign orgs or umapis and how the sync works in that case.